README.md

```diff
@@ -79,11 +83,24 @@ See [Install Docker Compose](https://docs.docker.com/compose/install/) for detai
    chown -R root docker/etc/cron.d
    ```
 
-8. Launch the containers. It is recommeded to wait 1 minute in order for each container to configure fully before launching the next
+8. Launch the containers.
+
+- With Docker Compose:
+
+   It is recommeded to wait 1 minute in order for each container to configure fully before launching the next
    ```
    docker-compose -f yaml/docker-compose.yaml up -d --force-recreate apel_mysql
+   # Wait 1 minute
    docker-compose -f yaml/docker-compose.yaml up -d --force-recreate apel_server
+   # Wait 1 minute
    docker-compose -f yaml/docker-compose.yaml up -d --force-recreate apel_rest_interface
    ```
 
+- With Ansible:
+
+   First, replace `<working directory>` in `yaml/ansible.yaml` with the absoulte file path of inflated source code, then run the below command.
```

Comment:
"absolute"

"inflated source code"?

I prefer "command below"

Reply:
changes made.

`README.md` says to download and unzip the source code as it contains schemas and yaml files needed for deploying via docker. And 'inflating' is the the verb unzip uses.

Reply:
I'd rather keep it more generic then: "decompressed"

Reply:
chnaged